src/components/Forward/Forward.js

```diff
@@ -0,0 +1,30 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import classNames from 'classnames'
+
+const propTypes = {
+  className: PropTypes.string,
+  scrollToRef: PropTypes.oneOfType([
+    PropTypes.func,
+    PropTypes.shape({ current: PropTypes.instanceOf(Element) })
+  ])
+}
+
+const Forward = ({ className, scrollToRef, ...attributes }) => {
+  const classes = classNames(className, 'forward')
+  return (
+    <a
+      className={classes}
+      onClick={() =>
+        scrollToRef.current.scrollIntoView({
```

Comment:
What happens when `scrollToRef` is a function here?

Reply:
scrollToRef will be a callback function for a ref.
https://reactjs.org/docs/refs-and-the-dom.html#callback-refs

Reply:
If I understood it correctly, that's a callback to get the `ref`.
Here instead you are using the `ref` to scroll the view to it: of the two `PropTypes` definitions for it, I see only the `shape` one that fits this code. If I pass a callback to this component as `scrollToRef` the `onClick` function I guess wilol throw and error, no?

Reply:
Ah yes it doesn't work, it doesn't throw an error as such but the callback ref should be handled differently